### PR TITLE
Documentation: Add .isQuaternion and .isTexture

### PR DESCRIPTION
**Pull request by zarov**
They were missing from the documentation.

### CODE REVIEW

File: docs/api/zh/math/Quaternion.html [hunk 1]
```diff
@@ -44,6 +44,13 @@ <h3>[name]( [param:Float x], [param:Float y], [param:Float z], [param:Float w] )
 
 		<h2>Properties</h2>
 
+		<h3>[property:Boolean isQuaternion]</h3>
+		<p>
+			Used to check whether this or derived classes are Quaternions. Default is *true*.<br /><br />
```
Comment (Mugen87) [inline docs/api/zh/math/Quaternion.html]: We usually use `TODO` as a placeholder so a Chinese community member can easily find missing parts and translate them.

File: docs/api/zh/textures/Texture.html [hunk 2]
```diff
@@ -54,6 +54,14 @@ <h3>[property:Image image]</h3>
 		并在视频播放时不断地更新这个纹理贴图。——[page:VideoTexture VideoTexture] 类会对此自动进行处理。
 		</p>
 
+		<h3>[property:Boolean isTexture]</h3>
+		<p>
+			用于测试这个类或者派生类是否为Texture，默认为*true*。<br /><br />
```
Comment (Mugen87) [inline docs/api/zh/textures/Texture.html]: Have you already translated this? ;)

Comment (zarov) [reply docs/api/zh/textures/Texture.html]: I don't speak chinese, but I copied it from another documentation page, assuming it is the same everywhere for this property ? Anyway I can remove chinese translations if you wish.

File: docs/api/zh/math/Quaternion.html (hunk 1 above)
Comment (zarov) [reply docs/api/zh/math/Quaternion.html]: Ok, I can change that, I assumed it was needed to be in english at least, viewing that the entire file is in english as well.

Comment (zarov) [reply docs/api/zh/math/Quaternion.html]: It can even be in chinese directly in fact, if you validate the other translation.

Comment (Mugen87) [reply docs/api/zh/math/Quaternion.html]: Oh, I did not see in the preview that the rest of the file is not yet translated. You can leave the text as it is 😉

File: docs/api/zh/textures/Texture.html (hunk 2 above)
Comment (Mugen87) [reply docs/api/zh/textures/Texture.html]: I've translated the text out of curiosity with google-translate and it seems to be fine. 

In general, I would avoid doing copy/paste and let the Chinese devs handle the translation.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**